content/posts/newsletter-007/index.md

```diff
@@ -553,6 +562,16 @@ For more details, please see the [`specs` changelog].
 [specs]: https://crates.io/crates/specs
 [`specs` changelog]: https://github.com/amethyst/specs/blob/0.16.1/CHANGELOG.md#0161-2020-02-18
 
+### [Skulpin][skulpin]
```

Comment:
Skulpin was already featured in 4th newsletter: https://rust-gamedev.github.io/posts/newsletter-004/#skulpin

Are there any more recent updates?

Reply:
Just SDL2 support and support for the latest version of winit, we can leave it out

Reply:
(sorry! 🙇)